app/src/main/java/com/github/se/polyfit/ui/flow/AddMealFlow.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.se.polyfit.ui.flow
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.github.se.polyfit.ui.navigation.Navigation
+import com.github.se.polyfit.ui.navigation.Route
+import com.github.se.polyfit.ui.screen.IngredientScreen
+import com.github.se.polyfit.ui.screen.NutritionScreen
+import com.github.se.polyfit.viewmodel.meal.MealViewModel
+
+@Composable
+fun AddMealFlow(
+    mainNavigation: Navigation,
+    userID: String,
+    mealViewModel: MealViewModel = MealViewModel(userID)
```

Comment:
if previous modif is accepted
```suggestion
fun AddMealFlow(
    goBack: () -> Unit,
    navigateToNutrition: () -> Unit,
    userID: String,
    mealViewModel: MealViewModel = MealViewModel(userID)
```